sqlx-sqlite/Cargo.toml

```diff
@@ -44,6 +43,8 @@ tracing = { version = "0.1.37", features = ["log"] }
 serde = { version = "1.0.145", features = ["derive"], optional = true }
 regex = { version = "1.5.5", optional = true }
 urlencoding = "2.1.3"
+tokio = { workspace = true }
+async-channel = "2.2.0"
```

Comment:
Where do you depend on this?

Reply:
Hmm looks like a dependency of previous implementation. Removing it, does not seem necessary.

Reply:
Removed: https://github.com/madsim-rs/sqlx/pull/3/files#r1818415114